parcels/field.py

```diff
@@ -160,14 +159,12 @@ def __init__(
         allow_time_extrapolation: bool | None = None,
         gridindexingtype: GridIndexingType = "nemo",
         to_write: bool = False,
-        **kwargs,
+        data_full_zdim=None,
```

Comment:
Do we really need this `data_full_zdim`? It's for B-grid interpolation, but that should (later?) be incorporated in the Interpolator? So remove now? Or do you want to remove it in a later PR?

Reply:
I can make another PR